Fika.Core/FikaPlugin.cs

```diff
@@ -136,6 +137,7 @@ public class FikaPlugin : BaseUnityPlugin
         public static ConfigEntry<bool> PingUseOpticZoom { get; set; }
         public static ConfigEntry<bool> PingScaleWithDistance { get; set; }
         public static ConfigEntry<float> PingMinimumOpacity { get; set; }
+        public static ConfigEntry<EPingSound> PingSound { get; set; }
```

Comment:
```suggestion
        public static ConfigEntry<EUISoundType> PingSound { get; set; }
```
Could you not use the enum here, this way you don't need the helper function?

Reply:
That definitely works, but I didn't want to flood the config menu with a bunch of sounds like `PlayerIsDead` or `GeneratorTurnOn` which play a bit too long to be appropriate for a 'pingy' sound. I can definitely change it if you feel it's better to provide all options though.

Reply:
> That definitely works, but I didn't want to flood the config menu with a bunch of sounds like `PlayerIsDead` or `GeneratorTurnOn` which play a bit too long to be appropriate for a 'pingy' sound. I can definitely change it if you feel it's better to provide all options though.
> 
> Is there a way to limit the options for EUISoundType?

I never opened the editor and forgot that these existed. Good point!